src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsuserpreferences/controller/CourtsController.kt

```diff
@@ -19,6 +20,8 @@ class CourtsController {
 
   @ResponseStatus(value = HttpStatus.CREATED)
   @PutMapping("/users/{uuid}/preferences/courts")
-  fun updatePreferences(@PathVariable uuid: UUID, @RequestBody user: Users): List<String> =
-    repository.save(Users(uuid, user.courts)).courts
+  fun updatePreferences(@PathVariable uuid: UUID, @RequestBody preference: CourtsDTO): List<String> {
```

Comment:
Sorry Paul, not sure why we need "CourtsDTO". I haven't been close to this but I expected generic JSON (like the database table), along the lines of 
{
  user_id : "chris",
  key: "courts",
  values: [
    "B10JQ", 
    "B20XX"
  ]
}